Added support for Smarthjemmet.dk QUAD-ZIG-SW 4 button remote

Nicolai- opened the pull request:
Added support for Smarthjemmet.dk QUAD-ZIG-SW 4 button remote (https://www.zigbee2mqtt.io/devices/QUAD-ZIG-SW.html)

Thank you for taking the time to work on a Pull Request. Your contribution is really appreciated! :tada:
**Please don't delete any part of the template**, since keeping the provided structure will help maintainers to review your work more rapidly.

Sections marked as \* are required and need to be filled in.

## Proposed change\*

Added Controller for Smarthjemmet.dk QUAD-ZIG-SW 4 button remote and implemented support in light hook
The documentation part is untested.

## Checklist\*

- [x] I followed sections of the [Contribution Guidelines](https://github.com/EPMatt/awesome-ha-blueprints/blob/main/CONTRIBUTING.md) relevant to changes I'm proposing.
- [x] I properly tested proposed changes on my system and confirm that they are working as expected.
- [x] I formatted files with Prettier using the command `npm run format` before submitting my Pull Request.

Nicolai- committed:
Added support for Smarthjemmet.dk QUAD-ZIG-SW 4 button remote (https://www.zigbee2mqtt.io/devices/QUAD-ZIG-SW.html)

yarafie commented:
@Nicolai- 

https://yarafie.github.io/awesome-ha-blueprints/docs/blueprints/controllers/smarthjemmet_dk_quad_zig_sw

Nicolai- commented:
> @Nicolai-
> 
> https://yarafie.github.io/awesome-ha-blueprints/docs/blueprints/controllers/smarthjemmet_dk_quad_zig_sw

Thanks, I will try to look into it setting it up again, so I can see the docs.

Nicolai- commented:
I cleaned up a bit and started using @yarafie aproach about implicit choosing the integration based on trigger ids from #693

Also removed everything relating to "Last Controller Event" as this is not needed for this controller as the double/tripple press is not a virtual event for this controller

yarafie commented:
> I cleaned up a bit and started using @yarafie aproach about implicit choosing the integration based on trigger ids from #693
> 
> Also removed everything relating to "Last Controller Event" as this is not needed for this controller as the double/tripple press is not a virtual event for this controller

Updated my fork with your latest version.

EPMatt committed:
Merge branch 'main' into ns_smarthjemmet_dk_quad_zig_sw
